Updated release notes

cs-abhishek-shukla opened the pull request.

cs-abhishek-shukla committed:
Updated release notes

cs-rohit-khune commented:
@cs-abhishek-shukla 

- Release notes look good to me
- I made a few changes in [docs/usage.md] as well, please make sure those changes are also reviewed

cs-abhishek-shukla committed:
1.0.2-changes
- Rewrote the usage section
- Added screenshots
- Row style in setup
- Rewrote release notes

cs-rohit-khune commented on docs/usage.md:
```diff
@@ -19,42 +19,63 @@ To add this widget to the detail view of a module record, perform the following
+    ![](./media/alert-detailed-view.png)
 
-3. Click **Add Widget** and select the **Fields Of Interest** widget.
+2. Click **Edit Template** (![](./media/icon-edit.svg)) to display its System View Template (SVT).
```
the icon-edit.svg here dose not display properly in dark theme of git. 

<img width="614" alt="image" src="https://github.com/fortinet-fortisoar/widget-fields-of-interest/assets/46894140/64b1cf5d-5e39-4ab7-a0c7-76f5e134a4ee">

Please try add some background to make it visible

cs-abhishek-shukla replied:
Oh!. Thanks. Will do

cs-rohit-khune commented on docs/usage.md:
```diff
@@ -19,42 +19,63 @@ To add this widget to the detail view of a module record, perform the following
+
+14. Click **Apply Changes** to add the widget to the detail view of the module's record.
 
 ### Fields of Interest Widget View Panel Views
```
In `Fields of Interest Widget View Panel Views` section, can we add both screenshots from the same alert, just to maintain consistency?

cs-abhishek-shukla replied:
I thought I used the same alert. I will check and fix

cs-abhishek-shukla committed:
Review comments
- Added a background to the icons used
- Added a new tag to the new field
- Fixed the screenshot to be from the same alert
- Edited the first line of release notes

cs-rohit-khune reviewed (approved):
Approved
